extensions/ql-vscode/src/common/interface-types.ts

```diff
@@ -577,6 +577,11 @@ interface SetModeledMethodMessage {
   method: ModeledMethod;
 }
 
+interface SetMultipleModeledMethodsMessage {
```

Comment:
👍🏼 

Adding a new type makes sense to me. Since we're reusing `SetModeledMethodMessage` in a few places, we'd have to change them all at once.

Once we're fully done with the migration, we can rename this type if we want to.